### DISCUSSION TITLE
New citations to the 3W Project itself and/or its main resources

### DISCUSSION THREAD

**Discussion by ricardoevvargas** [petrobras/3W#3] [General]
**Do you know any other published work (not covered in [this document](https://github.com/Petrobras/3W/blob/master/LIST_OF_CITATIONS.md)) that cites the 3W Project itself and/or its main resources?**

If yes, please let us know by adding a comment here in this discussion.

Thanks for the contribution!

**Comment by AndrePauloFM** [comment]
This paper, using 3W dataset, has just been published in the **Journal of Petroleum Science and Engineering**. (published August 2022) 
**"Improving performance of one-class classifiers applied to anomaly detection in oil wells"**
The proposed methodology improves the performance of LSTM autoencoder and one-class SVM for anomaly detection in oil wells.
more details visit: https://www.sciencedirect.com/science/article/abs/pii/S0920410522008348

**Reply by ricardoevvargas** [reply]
Hi @AndrePauloFM.

We appreciate your contribution. This paper will be added to our [LIST_OF_CITATIONS.md](https://github.com/petrobras/3W/blob/master/LIST_OF_CITATIONS.md) soon.

**Reply by ricardoevvargas** [reply]
This paper was added with the PR #24. @AndrePauloFM, congrats on this publication 👏👏👏

**Comment by sergiolimanetto** [comment]
This paper, citing 3W dataset, has just been published in Applied Computational Intelligence and Soft Computing
Volume 2022, Article ID 1558381 (https://doi.org/10.1155/2022/1558381)
"Anomaly Detection Using Explainable Random Forest for the Prediction of Undesirable Events in Oil Wells".

**Reply by ricardoevvargas** [reply]
Hi @sergiolimanetto.

We appreciate your contribution. This paper will be added to our [LIST_OF_CITATIONS.md](https://github.com/petrobras/3W/blob/master/LIST_OF_CITATIONS.md) soon.

**Reply by ricardoevvargas** [reply]
This paper was added with the PR #25. @sergiolimanetto, if you identify other papers, master's degree dissertations and doctoral theses not listed [here](https://github.com/Petrobras/3W/blob/master/LIST_OF_CITATIONS.md), please comment here in this discussion. 👍

**Comment by cjmunaro** [comment]
Two papers were recently published using or citing 3W dataset.

Gatta, F., Giampaolo, F., Chiaro, D., & Piccialli, F. (2022). Predictive maintenance for offshore oil wells by means of deep learning features extraction. Expert Systems, e13128.
https://doi.org/10.1111/exsy.13128

In this paper a 1D Convolutional Neural Network is used to extract features from raw data using windows of different lengths. The features are then fed into different machine learning algorithms.

Melo, A., Câmara, M. M., Clavijo, N., & Pinto, J. C. (2022). Open benchmarks for assessment of process monitoring and fault diagnosis techniques: a review and critical analysis. Computers & Chemical Engineering, 107964.
https://doi.org/10.1016/j.compchemeng.2022.107964

This paper presents and discusses datasets and simulators for testing of process monitoring and fault diagnosis techniques. A exploratory analysis of the 3W dataset provides an overview of the  diversity of data that can be helpful to propose fault diagnosis techniques.

**Reply by ricardoevvargas** [reply]
Hi @cjmunaro. 

What doubly good news. Thank you for letting us know. Both will be added to our [LIST_OF_CITATIONS.md](https://github.com/petrobras/3W/blob/master/LIST_OF_CITATIONS.md) soon.

**Reply by ricardoevvargas** [reply]
These papers (along with a 3rd one) were added with the PR #28. @cjmunaro, we appreciate your contribution. 👍

**Comment by yagomsv** [comment]
This article is citing the application of models based on RNN using the 3w sataset. Article published in March 2023 in the magazine "IEEE Intelligent Systems".
https://ieeexplore.ieee.org/document/10058896

**Reply by ricardoevvargas** [reply]
Hi, @yagomsv. 

Thank you for letting us know that. 

I realized that in this work the 3W was used intensively. So this article can be inspirational to all of us. 

I will update the [citation list to 3W](https://github.com/petrobras/3W/blob/main/LIST_OF_CITATIONS.md) ASAP.

**Reply by ricardoevvargas** [reply]
Done with PR #75. @yagomsv, we appreciate your contribution. 👍

**Comment by Ilanfigueiredo** [comment]
My PhD thesis used the 3W database to validate the proposed new self-learning approach. The thesis is available in the institutional repository of SENAI CIMATEC: http://repositoriosenaiba.fieb.org.br/handle/fieb/1730

**Reply by ricardoevvargas** [reply]
Hi, @Ilanfigueiredo. 

Congratulations on obtaining the PhD degree and for this work of yours, which is a relevant contribution to 3W. Your thesis, including the link provided, was added in the list of citations with the PR #85.

**Comment by afraniomelo** [comment]
In my doctoral thesis, I conducted an in-depth exploratory data analysis on multiple benchmarks, one of which was the 3W Dataset. The thesis can be accessed through [this link](https://sucupira.capes.gov.br/sucupira/public/consultas/coleta/trabalhoConclusao/viewTrabalhoConclusao.xhtml?popup=true&id_trabalho=13725906).

**Reply by ricardoevvargas** [reply]
Hi, @afraniomelo.

That is interesting. I believe 3W users can be inspired by your work. I will add your thesis on the [citation list to 3W](https://github.com/petrobras/3W/blob/main/LIST_OF_CITATIONS.md) ASAP.

**Reply by ricardoevvargas** [reply]
Hi, @afraniomelo.

Done with PR #87. We appreciate your contribution. 👍

**Comment by ricardoevvargas** [comment]
Hi. Since our last check, we have detected:

- Citations in 4 articles recently published in journals (Journal of Process Control, Evolving Systems, IEEE Access and Chemical Engineering Transactions);
- Citaiton in 1 preprint article that was recently published in a journal (Energy).

Therefore, as far as we know, the 3W dataset has been cited by **67 works**, which establish a framework that can be useful to us in various ways. Complete list to be consulted whenever important is [here](https://github.com/petrobras/3W/blob/main/LIST_OF_CITATIONS.md).

**IMPORTANT**: remember that everyone can indicate here in this discussion channel additional papers, final graduation project, master's degree dissertations, doctoral theses that cite the 3W dataset. We appreciate all kinds of contributions!

**Comment by ricardoevvargas** [comment]
Hello, everyone.

@AndrePauloFM, Celso Munaro, Patrick Ciarelli and I are happy to announce that our [article](https://doi.org/10.1016/j.eswa.2023.122895) exploring 3W features was recently published by the journal Expert Systems with Applications. We hope it will inspire you. 

I've just updated our list of publications that reference 3W: https://github.com/petrobras/3W/blob/main/LIST_OF_CITATIONS.md.

**Comment by sergiolimanetto** [comment]
Hello Everyone.

This paper has just been published describing a modular system for developing fault detection & classification AI models for the 3W dataset. The MAIS systems is available in the Open Lab initiative by Petrobras. In addition, we investigate the use oh wavelet-based attributes to model the different time dynamics of distinct failure classes within the 3W. 

T. L. B. Dias, M. A. Marins, C. L. Pagliari, R. M. E. Barbosa, M. L. R. de Campos, E. A. B. Silva, and S. L. Netto, “Development of oil-well fault classifiers using a wavelet-based multivariable approach in a modular architecture,” Society of Petroleum Engineers Journal, DOI: https://doi.org/10.2118/221463-PA

Congratulations to all the success achieved so far by the 3W dataset/initiative. Regards,
sergio

**Reply by ricardoevvargas** [reply]
Hi, @sergiolimanetto. Thank you for letting us know about this recent publication. I will update the [citation list to 3W](https://github.com/petrobras/3W/blob/main/LIST_OF_CITATIONS.md) ASAP.

It can be noted that the exploitation of the 3W dataset in this work was intense, which is great. In particular, I think the use of Wavelets is very interesting. So this article can be inspirational to all of us.

Congratulations on this important publication.

**Reply by ricardoevvargas** [reply]
Hi, @sergiolimanetto.

Done with PR #106. We appreciate your contribution. 👍

**Comment by gfidelisp** [comment]
Hello, everyone. 

Here is our paper published in ENCIT 2024 - 20th Brazilian Congress of Thermal Sciences and Engineering, 2024.

Villamil, R. H.; May, J. S.; Fallgater, R. H.; Vargas, R.; Nakashima, A. T. D.; Peixer, G.f.; Lozano, J. A.; Barbosa Jr., J. R. Assessment Of Deep Learning Techniques For Anomaly Detection In Offshore Oil Wells. In: ENCIT 2024 - 20th Brazilian Congress of Thermal Sciences and Engineering, 2024, Foz do Iguaçu.

Kind regards!

**Reply by ricardoevvargas** [reply]
Hi, @gfidelisp.

Done with PR #133. We appreciate your contribution. 👍 

Now our list is made up of **almost 100 works** that cite 3W.

**Comment by ramdhan1989** [comment]
Hi,
this is our paper presented in SPE ATCE 2025. 
Wibawa, R. A., Wang, M., and B. Jha. "Exploring Modern Feature Extraction Techniques for Improved Offshore Fault Detection in Oil and Gas Operations." Paper presented at the SPE Annual Technical Conference and Exhibition, Houston, Texas, USA, October 2025. doi: https://doi-org.libproxy2.usc.edu/10.2118/228061-MS

Best regards

**Reply by ricardoevvargas** [reply]
Hi, @ramdhan1989. Thank you for letting us know about this recent publication. We will update the [citation list to 3W](https://github.com/petrobras/3W/blob/main/LIST_OF_CITATIONS.md) ASAP.

Congratulations on this important publication.

**Reply by ricardoevvargas** [reply]
Hi, @ramdhan1989.

Done with PR #176. We appreciate your contribution. 👍

Now our lists are made up of 121works that cite 3W.

**Comment by LuizAlbino** [comment]
Hi, I used 3W dataset on my Final Graduation Projects

Here is the repo we used: https://github.com/LuizAlbino/final_gradution_project

**Reply by ricardoevvargas** [reply]
Hi, @LuizAlbino.

It is gratifying to know that the 3W Dataset was useful during your graduation course and that we are contributing to the professional development of experts and the creation of new digital solutions and startups in the field of oil well monitoring.

On behalf of the 3W Community, I thank you for your contribution.
